packages/manager/apps/pci-savings-plan/src/components/CreatePlanForm/CreatePlanForm.tsx

```diff
@@ -528,7 +532,11 @@ export const CreatePlanFormContainer = ({
   ];
 
   const instancesInformation = useMemo(
-    () => getInstancesInformation(technicalList, t),
+    () =>
+      getInstancesInformation(t).map((i) => ({
+        ...i,
+        technical: technicalList,
```

Comment:
Is this required ? This would repeat a lot of times the information while it does not depend on the object